src/Xamarin.Android.Build.Tasks/Utilities/ManifestDocument.cs

```diff
@@ -102,14 +102,22 @@ internal class ManifestDocument
 			}
 		}
 		public string GetMinimumSdk () {
+			log.LogWarning ("GetMinimumSdk called");
```

Comment:
We're not going go want to keep these `LogWarning()` calls in the final merged commit. :-)

Reply:
I know, but I can't repro it locally, there's nothing in the logs so I need to get the information somehow...